Command/ExpiredVideoDeleteCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Pumukit\ExpiredVideoBundle\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+class ExpiredVideoDeleteCommand extends ContainerAwareCommand
+{
+    private $dm = null;
+    private $mmobjRepo = null;
+    private $days = 365;
+    private $user_code = "owner";
```

Comment:
@rubenrua, shouldn't these two parameters be configurable? $days and $user_code defined in DependencyInjection/Configuration.php?